libs/mock-block-dock/src/service-module-callbacks.ts

```diff
@@ -56,8 +61,15 @@ const callExternalApiMethod = async (params: {
             data && typeof data === "object" && "errors" in data
               ? (data.errors as any)?.[0]?.message
               : "An unknown error occurred.",
+          // @ts-expect-error –– @todo why is this an error
```

Comment:
Trying to figure this out too, I think it's due to the conditional type inside the return value in core. The conditionals there have caused problems before 🙃 

While looking into it I noticed we have:
- `GraphEmbedderMessages` _and_
- `GraphBlockMessageCallbacks`

But only
- `ServiceEmbedderMessageCallbacks`

Any idea why?

Reply:
@Alfred-Mountfield going to do this as a drive-by as part of https://github.com/blockprotocol/blockprotocol/pull/1114, apologies I wasn't aware we had a `MessageReturn`